libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -5104,12 +5104,17 @@ MAV_RESULT GCS_MAVLINK::handle_command_int_packet(const mavlink_command_int_t &p
     case MAV_CMD_DO_SET_MODE:
         return handle_command_do_set_mode(packet);
 
+    case MAV_CMD_DO_SET_ROI_NONE: {
```

Comment:
I'm not really a fan of moving handling of this command up into the GCS layer.  That seems like we're moving in the wrong direction.

I think the inconsistently comes from how Copter's special handling of ROI in that we want to be able to point the copter's nose at the ROI target in case the gimbal does not have yaw control.

Reply:
Agreed, but moving it here is consistent with how `DO_SET_ROI` and `DO_SET_ROI_LOCATION` are currently handled.

I'd rather move this here now, then refactor all three of them together.